crates/composer_api/src/lib.rs

```diff
@@ -20,8 +20,33 @@ pub enum Event {
     LogStats(LogStats),
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub enum LogStats {
+    Aggregate(AggregateLogStats),
+    Individual(IndividualLogStats),
+}
+
+/// Each log carries its own timestamp.
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub struct IndividualLogStats {
+    // We use duration since the UNIX epoch as a serializable timestamp.
+    pub logs: Vec<(Duration, LogLevel)>,
+}
+
+// FIXME: Duplicates the `log` crate definitions, but it's likely
+// still better than pulling the dependency.
```

Comment:
Agree. Alternatively, we don't necessarily have to have 1:1 mapping. For our purposes just two levels eg. `Info` and `Error` mapped from `Trace - Info` and `Warn - Error` respectively may be enough. But we'll see when implementing the sound effects for this probe (which I plan to do shortly)